web/content/js/nav.js

```diff
@@ -67,15 +61,57 @@ let navItems = [
 		href: "/Application_Centric_Deployment/index-foreman-el.html"
 	}]
 },{
-	title: "Katello documentation", items: [
+	title: "Foreman on Debian", items: [
+	{
+		title: "Planning Guide",
+		href: "/Planning_Guide/index-foreman-deb.html"
```

Comment:
https://github.com/theforeman/foreman-documentation/pull/679

Reply:
I actually moved this only to Katello (after I tipped the screenshots). When this is ready, we can put it back.

Reply:
I see this is indeed outdated. Marking as resolved.